class/openshift-prometheus-proxy.yml

```
@@ -1,5 +1,9 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: https
+        source: https://raw.githubusercontent.com/appuio/openshift-prometheus-proxy/${openshift_prometheus_proxy:version}/template/openshift-prometheus-proxy.yaml
+        output_path: dependencies/openshift-prometheus-proxy/manifests/template.yaml
```

```suggestion
        output_path: dependencies/openshift-prometheus-proxy/manifests/${openshift_prometheus_proxy:version}/template.yaml
```

Otherwise already downloaded manifests won't be overwritten by new versions.